app/src/main/java/org/merakilearn/ui/onboarding/WelcomeFragment.kt

```diff
@@ -2,16 +2,24 @@ package org.merakilearn.ui.onboarding
 
 import android.content.Context
 import android.os.Build
+import android.content.Intent
 import android.os.Bundle
+import android.util.Log
```

Comment:
Use timber instead of logs

Reply:
Any specific reason for using timber ?

Reply:
It doesn't print logs in production and also since we are already using it in chatSDK so its not extra burden